protocol/x/clob/module.go

```diff
@@ -177,6 +177,16 @@ func (am AppModule) PreBlock(ctx context.Context) (appmodule.ResponsePreBlock, e
 	}, nil
 }
 
+// BeginBlock executes all ABCI BeginBlock logic respective to the clob module.
+func (am AppModule) Precommit(ctx context.Context) error {
```

Comment:
Callback was not previously implemented, so `ClobModule.Precommit` was not being run